lib/ProductOpener/Display.pm

```diff
@@ -435,6 +435,33 @@ sub process_template($template_filename, $template_data_ref, $result_content_ref
 }
 
 
+=head2 redirect($status_code, $redirect_url)
```

Comment:
didn't you forget to add it to export ?

Reply:
right, I'll add it. I just noticed I was also exporting init which has been renamed to init_request